examples/transfer_position.py

```diff
@@ -1,7 +1,242 @@
+    )
+
+    position_amount = position_amount.quantize(Decimal("0.1"), rounding=ROUND_HALF_UP)
+    position_price = position_price.quantize(Decimal("0.1"), rounding=ROUND_HALF_UP)
+
+    print(f"Position amount: {position_amount}")
+    print(f"Position average price: {position_price}")
+
+    ###################
+    # Define Transfer #
+    ###################
+
+    # Transfer amount and price from position data, with 0 fees for transfers
+    transfer_amount = abs(position_amount)  # Use absolute value for transfer amount
+    transfer_price = position_price
+    max_fee = Decimal("0")  # No fees for transfers
+
+    # Determine maker direction - maker must reduce their position
+    # If position is positive, maker sells (reduces long position)
+    # If position is negative, maker buys (reduces short position)
+    original_position_amount = Decimal(position["amount"])
+    if original_position_amount > 0:
+        maker_is_bid = False  # Sell to reduce long position
+        maker_direction = "sell"
+        taker_is_bid = True  # Taker buys
+        taker_direction = "buy"
+    else:
+        maker_is_bid = True  # Buy to reduce short position
+        maker_direction = "buy"
+        taker_is_bid = False  # Taker sells
+        taker_direction = "sell"
+
+    print(f"Original position amount: {original_position_amount}")
+    print(f"Maker direction: {maker_direction} (reducing position)")
+    print(f"Taker direction: {taker_direction}")
+
+    print("Creating transfer-specific signed actions...")
+
+    # Create maker order parameters
+    maker_nonce = utils.get_action_nonce()
+    maker_signature_expiry = utils.MAX_INT_32
+
+    # Create taker order parameters - ensure different nonce
+    time.sleep(0.001)  # Small delay to ensure different timestamp
+    taker_nonce = utils.get_action_nonce()
+    taker_signature_expiry = utils.MAX_INT_32
+
+    maker_action = SignedAction(
+        subaccount_id=FROM_SUBACCOUNT_ID,
+        owner=DERIVE_CONTRACT_WALLET_ADDRESS,
+        signer=session_key_wallet.address,
+        signature_expiry_sec=maker_signature_expiry,
+        nonce=maker_nonce,
+        module_address=TRADE_MODULE_ADDRESS,
+        module_data=TradeModuleData(
+            asset_address=instrument["base_asset_address"],
+            sub_id=int(instrument["base_asset_sub_id"]),
+            limit_price=transfer_price,
+            amount=transfer_amount,
+            max_fee=max_fee,
+            recipient_id=FROM_SUBACCOUNT_ID,
+            is_bid=maker_is_bid,
+        ),
+        DOMAIN_SEPARATOR=DOMAIN_SEPARATOR,
+        ACTION_TYPEHASH=ACTION_TYPEHASH,
+    )
+
+    taker_action = SignedAction(
+        subaccount_id=TO_SUBACCOUNT_ID,
+        owner=DERIVE_CONTRACT_WALLET_ADDRESS,
+        signer=session_key_wallet.address,
+        signature_expiry_sec=taker_signature_expiry,
+        nonce=taker_nonce,
+        module_address=TRADE_MODULE_ADDRESS,
+        module_data=TradeModuleData(
```

Comment:
Can we use create wrapper classes around  "TradeModuleData" -> e.g. "TakerTransferPositionModuleData" and "MakerTransferPositionModuleData". This way users don't need to realize they have to use TradeModuleData. 

Check out `TakerTransferPositionsModuleData` which wraps `RFQModuleData`.